core/src/main/java/org/apache/accumulo/core/metadata/schema/TabletsMetadata.java

```diff
@@ -233,6 +269,19 @@ public Options forTablet(KeyExtent extent) {
       return this;
     }
 
+    @Override
+    public Options forTablets(Collection<KeyExtent> extents) {
+      if (!extents.stream().map(e -> DataLevel.of(e.tableId()))
+          .allMatch(dl -> dl == DataLevel.USER)) {
```

Comment:
Instead of `! allMatch`, you can just use `anyMatch`:

```suggestion
      if (extents.stream().map(e -> DataLevel.of(e.tableId()))
          .anyMatch(dl -> dl == DataLevel.USER)) {
```

You can also split up the two steps in the map function to use method references. Sometimes this is easier to understand:

```suggestion
      if (extents.stream().map(KeyExtent::tableId).map(DataLevel::of)
          .anyMatch(dl -> dl == DataLevel.USER)) {
```

Reply:
These are not equivalent. Wrote the little test below.

```java
	var dlset = List.of(DataLevel.USER, DataLevel.USER, DataLevel.USER);
	System.out.println(!dlset.stream().allMatch(dl -> dl == DataLevel.USER));
	System.out.println(dlset.stream().anyMatch(dl -> dl == DataLevel.USER));
```  

prints

```
false
true
```

The following pattern is equivalent and uses anyMatch.  I think it makes the code longer so not sure its worthwhile.  Also not too worried about efficiency because the expected case is all USER, so normally it will iterator through all with anyMatch or allMatch.

```java
	EnumSet<DataLevel> unsupportedLevels = EnumSet.complementOf(EnumSet.of(DataLevel.USER));
	
	var dlset = List.of(DataLevel.USER, DataLevel.USER, DataLevel.USER);
	System.out.println(!dlset.stream().allMatch(dl -> dl == DataLevel.USER));
	System.out.println(dlset.stream().anyMatch(dl -> unsupportedLevels.contains(dl)));
	
	dlset = List.of(DataLevel.USER, DataLevel.ROOT, DataLevel.USER);
	System.out.println(!dlset.stream().allMatch(dl -> dl == DataLevel.USER));
	System.out.println(dlset.stream().anyMatch(dl -> unsupportedLevels.contains(dl)));
```

This prints 

```
false
false
true
true
```

Reply:
I thought of something shorter that uses anyMatch pushing that soon.

Reply:
Changed in 858ec8f